test/uk/gov/hmrc/ups/connectors/MockHttp.scala

```diff
@@ -29,6 +32,12 @@ trait MockHttpGet extends MockitoSugar {
   val http = new HttpGet with WSHttp {
     override def doGet(url: String)(implicit hc: HeaderCarrier) =
       Future.successful(httpWrapper.getF(url))
+
+    override protected def appNameConfiguration: Configuration = ???
+
+    override protected def actorSystem: ActorSystem = ActorSystem("test-actor-system")
```

Comment:
Do we need a new actor systems each time or could it be
private val anActorSystem = ActorSystem("test-actor-system")
..
override protected def actorSystem: ActorSystem = anActorSystem

Reply:
as far as I understand it wouldn't make any difference if I put it to the same trait

Reply:
fixed...